core/src/test/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectCreatorSpec.scala

```diff
@@ -2,7 +2,6 @@ package org.broadinstitute.dsde.rawls.billing
 
 import akka.http.scaladsl.model.headers.OAuth2BearerToken
 import bio.terra.profile.model.{AzureManagedAppModel, ProfileModel}
-import cromwell.client.ApiException
```

Comment:
I don't think this was the ApiException we intended to throw!

Reply:
Good catch